staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -613,6 +613,12 @@ func IsUnsupportedMediaType(err error) bool {
 	return false
 }
 
+// IsSSAErrorFailedToCreate checks if this error is due to SSA having failed to create a field manager for existing fields.
+func IsSSAErrorFailedToCreateManagerFromExistingFields(err error) bool {
+	message := err.Error()
+	return strings.HasPrefix(message, "failed to create manager for existing fields")
```

Comment:
relying on message parsing is brittle, can we make the errors in SSA part of the SSA API so we can use As and Is to detect them https://go.dev/blog/go1.13-errors ?

Reply:
Not yet, we need to find one and do it.

Reply:
Let me create an issue, we need to track this discussion separately.

Reply:
For xref: https://github.com/kubernetes/kubernetes/pull/118398